ansible_collections/arista/avd/plugins/action/validate_and_template.py

```diff
@@ -2,6 +2,9 @@
 
 __metaclass__ = type
 
+import cProfile
+import pstats
+
```

Comment:
Should this be committed?

Reply:
yes. It is just to add performance tooling to this plugin like we have it in yaml_templates_to_facts. Only used during dev/troubleshooting